### ISSUE TITLE
Unable to sign an Application

### ISSUE THREAD

**Issue by kaleb-g** [nowsecure/node-applesign#139]
Good day, I was trying to resign an application DVIA. 
MacOS 12.4 M1
applesign version 4.0.0

Here is the switches that I used 
```
applesign -i 749AD577AB7889A809D075F444A1B27F4649063D -m aa799d52-56aa-4c8b-964e-b31654b9bf8b.mobileprovision -a DVIA-v2-swift.ipa -c
```

Error:

**"Error: stdout:
stderr: Failed to parse entitlements: AMFIUnserializeXML: syntax error near line 12"**
````
swiftos.dylib","/Users/user/Downloads/Sample/DVIA-v2-swift.ipa.da820bec-31fb-47db-92fd-7582398cb558/Payload/DVIA-v2.app/Frameworks/libswiftsimd.dylib"]
Orphan: ["/Users/user/Downloads/Sample/DVIA-v2-swift.ipa.da820bec-31fb-47db-92fd-7582398cb558/Payload/DVIA-v2.app/libswiftRemoteMirror.dylib"]
Cleaning up /Users/user/Downloads/Sample/DVIA-v2-swift.ipa.da820bec-31fb-47db-92fd-7582398cb558
Error: stdout:
stderr: Failed to parse entitlements: AMFIUnserializeXML: syntax error near line 12
````

Not sure what I'm missing here.

But when I downgrade to version 3.9.3, the applesign is working.
Any help will do! TIA

**Comment by trufae** [comment]
Thanks for reporting!

I have investigated the issue and wrote a fix for it. Seems like the latest plist parser from Apple is a bit more strict. Read this for some context https://github.com/electron/osx-sign/issues/218 the regression was introduced after updating the plist dependency which had some different default settings that make `codesign` tool unhappy.

I just released applesign 4.0.1 with the fix.

**Comment by kaleb-g** [comment]
> Thanks for reporting!
> 
> I have investigated the issue and wrote a fix for it. Seems like the latest plist parser from Apple is a bit more strict. Read this for some context [electron/osx-sign#218](https://github.com/electron/osx-sign/issues/218) the regression was introduced after updating the plist dependency which had some different default settings that make `codesign` tool unhappy.
> 
> I just released applesign 4.0.1 with the fix.

Thank you for the quick fix as always!
